### PR TITLE
[yourporn] Drop support for yourporn.sexy and fix support for sxyprn.com

### PR DESCRIPTION
**Pull request by v-delta**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

https://yourporn.sexy doesn't exist anymore since a couple of months, but all the content is now available under https://sxyprn.com. For a period of time the [yourporn]-extractor worked for sxyprn.com anyways, but now it's broken (see #21645).

This PR drops support for the non-existing site yourporn.sexy and fixes the extraction of the `video_url` for sxyprn.com

### CODE REVIEW

File: youtube_dl/extractor/yourporn.py [hunk 1]
```diff
@@ -33,11 +32,13 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, video_id)
 
-        video_url = urljoin(url, self._parse_json(
-            self._search_regex(
-                r'data-vnfo=(["\'])(?P<data>{.+?})\1', webpage, 'data info',
-                group='data'),
-            video_id)[video_id]).replace('/cdn/', '/cdn5/')
+        url_parts = self._parse_json(self._search_regex(
+            r'data-vnfo=(["\'])(?P<data>{.+?})\1', webpage, 'data info', group='data'), video_id)[video_id].split("/")
```
Comment (dstftw) [inline youtube_dl/extractor/yourporn.py]: Carry long lines. Single quotes. Read coding conventions.

File: youtube_dl/extractor/yourporn.py [hunk 2]
```diff
@@ -33,11 +32,13 @@ def _real_extract(self, url):
...
+
+        aid = self._search_regex(r'data-aid=\'(.*?)\'', webpage, 'aid')
```
Comment (dstftw) [inline youtube_dl/extractor/yourporn.py]: Must allow both quotes kind. Do not capture empty string. Again read coding conventions.

**Comment by v-delta** [comment]
Sorry about that. Better?

**Comment by JChris246** [comment]
sure this method still works??

**Comment by v-delta** [comment]
Not anymore, no. Aparrently they've changed the generation of the video_url again.

**Comment by v-delta** [comment]
Looks like they put some sort of epoch into the video_url. [VideoDownloadHelper](https://www.downloadhelper.net/) gets the following video_url for this video

https://sxyprn.com/post/5d6fa4691538d.html

https://c9.trafficdeposit.com/vidi/h81x3100z91w8p0pzo9r07zp300/AmEkYvFTPynqOYkojoq1Fw/1567683260/5d5f0a215d13e/5d6fa4691538d.vid

1567683260 would be the actual epoch, but I wasn't able to fix this issue yet.

**Comment by v-delta** [comment]
Works now @dstftw

**Comment by dhwz** [comment]
is this still working? getting only invalid urls

**Comment by v-delta** [comment]
Any chance to get this merged? @dstftw

**Pull request closed without merging**